### ISSUE TITLE
Mention that elements with tooltips applied with for/forComponent API should be focusable

### ISSUE THREAD

**Issue by rolfsmeds** [vaadin/docs#2623]
When tooltips are applied to elements using the `for`/`forComponent` API, there is no guarantee that it will be announced by screen readers. The docs do mention that there is no a11y guarantee when used this way, but could give some pointers about how to give it better chance of being accessible, by either making the element focusable or giving it an appropriate role.

---

If you think this issue is important, add a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to help the community and maintainers prioritize this issue.
